multinode_integration_tests/tests/verify_bill_payment_utils/utils.rs

```diff
@@ -0,0 +1,1003 @@
+// Copyright (c) 2024, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use bip39::{Language, Mnemonic, Seed};
+use futures::Future;
+use itertools::Itertools;
+use lazy_static::lazy_static;
+use masq_lib::blockchains::chains::Chain;
+use masq_lib::utils::{derivation_path, NeighborhoodModeLight};
+use multinode_integration_tests_lib::blockchain::BlockchainServer;
+use multinode_integration_tests_lib::masq_node_cluster::MASQNodeCluster;
+use multinode_integration_tests_lib::masq_real_node::{
+    ConsumingWalletInfo, EarningWalletInfo, MASQRealNode, NodeStartupConfig,
+    NodeStartupConfigBuilder, PreparedNodeInfo,
+};
+use multinode_integration_tests_lib::utils::{open_all_file_permissions, UrlHolder};
+use node_lib::accountant::db_access_objects::payable_dao::{PayableDao, PayableDaoReal};
+use node_lib::accountant::db_access_objects::receivable_dao::{ReceivableDao, ReceivableDaoReal};
+use node_lib::accountant::gwei_to_wei;
+use node_lib::blockchain::bip32::Bip32EncryptionKeyProvider;
+use node_lib::blockchain::blockchain_interface::blockchain_interface_web3::{
+    BlockchainInterfaceWeb3, REQUESTS_IN_PARALLEL,
+};
+use node_lib::blockchain::blockchain_interface::lower_level_interface::{
+    LowBlockchainInt, ResultForBalance,
+};
+use node_lib::blockchain::blockchain_interface::BlockchainInterface;
+use node_lib::database::db_initializer::{
+    DbInitializationConfig, DbInitializer, DbInitializerReal, ExternalData,
+};
+use node_lib::sub_lib::accountant::PaymentThresholds;
+use node_lib::sub_lib::blockchain_interface_web3::transaction_data_web3;
+use node_lib::sub_lib::wallet::Wallet;
+use node_lib::test_utils;
+use node_lib::test_utils::standard_dir_for_test_input_data;
+use rustc_hex::{FromHex, ToHex};
+use std::cell::RefCell;
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+use std::thread;
+use std::time::{Duration, Instant, SystemTime};
+use tiny_hderive::bip32::ExtendedPrivKey;
+use web3::transports::Http;
+use web3::types::{Address, Bytes, SignedTransaction, TransactionParameters, TransactionRequest};
+use web3::Web3;
+
+pub type StimulateConsumingNodePayments = fn(&mut MASQNodeCluster, &MASQRealNode, &WholesomeConfig);
+
+pub type StartServingNodesAndLetThemPerformReceivablesCheck =
+    fn(&mut MASQNodeCluster, &WholesomeConfig) -> [MASQRealNode; 3];
+
+pub fn test_body(
+    test_inputs: TestInputs,
+    assertions_values: AssertionsValues,
+    stimulate_consuming_node_to_pay: StimulateConsumingNodePayments,
+    start_serving_nodes_and_activate_their_accountancy: StartServingNodesAndLetThemPerformReceivablesCheck,
+) {
+    // It's important to prevent the blockchain server handle being dropped too early
+    let (mut cluster, global_values, _blockchain_server) = establish_test_frame(test_inputs);
+    let consuming_node =
+        global_values.prepare_consuming_node(&mut cluster, &global_values.blockchain_interfaces);
```

Comment:
The second argument of this function is a reference to a field on `global_values`. Maybe it was intended or maybe it could be improved. Please take a look. 👀